client/components/notice/index.tsx

```diff
@@ -0,0 +1,146 @@
+/**
+ * External dependencies
+ */
+
+import React, { useEffect, FunctionComponent, useRef } from 'react';
+import classnames from 'classnames';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Internal dependencies
+ */
+import { TimerHandle } from 'calypso/types';
+/**
```

Comment:
Nit: We could leave an empty line here to separate those blocks properly:

```suggestion

/**
```